manager/integration/tests/conftest.py

```diff
@@ -58,14 +58,7 @@ def pytest_collection_modifyitems(config, items):
                 item.add_marker(skip_upgrade)
 
     using_csi = check_csi(core_api)
-    if using_csi:
-        skip_upgrade = pytest.mark.skip(reason="environment is not using " +
-                                               "flexvolume")
-        for item in items:
-            if "flexvolume" in item.keywords:
-                item.add_marker(skip_upgrade)
-
-    else:
+    if not using_csi:
```

Comment:
Error out if it's not CSI since it's the only driver we support now.

Reply:
removed the check, since we're only support csi driver now.